ansible_events/websocket.py

```diff
@@ -1,22 +1,29 @@
 import json
 import logging
+from asyncio.exceptions import CancelledError
 
 import websockets
 
+logger = logging.getLogger("ansible_events.websocket")
+
 
 async def send_event_log_to_websocket(event_log, websocket_address):
-    logger = logging.getLogger()
+    logger.info("websocket %s connecting", websocket_address)
     async for websocket in websockets.connect(websocket_address):
+        logger.info("websocket %s connected", websocket_address)
         event = None
         try:
             while True:
                 event = await event_log.get()
                 await websocket.send(json.dumps(event))
                 if event == dict(type="Shutdown"):
-                    break
+                    return
         except websockets.ConnectionClosed:
-            logger.warning(f"websocket {websocket_address} connection closed")
+            logger.warning("websocket %s connection closed", websocket_address)
             continue
```

Comment:
Since there is no logic below the `try: ... except:` block, the `continue` statements are redundant here.